### PR TITLE
Button and Dialog for Covered AT on DataManagement

### LINKED ISSUE #656: P1: Define categories of AT and show the covered category
_Related to #648: Covered AT Column Details._

We will define categories of AT and show the covered category, how many AT are covered within the category, and a button that popsup the list of them. For example, there could be a button named "4 Desktop Screen Readers" and activating it would open a dialog listing the screen readers.

### PR DESCRIPTION
**Pull request by stalgiag**
## Summary

This PR replaces the current listing of ATs in the Covered AT column on the Data Management page with a button that lists an inventory of covered ATs (simple now due to only supporting "Desktop Screenreaders" currently) and opens a dialog with all of the unique AT + AT version combos.

### Issues resolved

resolves #656

### CODE REVIEW

**Comment by stalgiag** [comment]
After a conversation with @alflennik, we decided that this list likely should be unique ATs not unique AT + AT Version combos. I will make this change on Wednesday when I am back on the project.

File: client/components/DataManagement/CoveredAtListDialogWithButton/index.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+const CoveredAtListDialogWithButton = ({ ats }) => {
+    const [show, setShow] = useState(false);
+    const handleShow = () => setShow(true);
+    const handleClose = () => setShow(false);
+
+    const renderButtonLabel = () => {
+        return (
+            <span>
+                <strong>{ats.length} Desktop</strong> Screenreaders
```
Comment (alflennik) [inline client/components/DataManagement/CoveredAtListDialogWithButton/index.jsx:15]: Would it be possible to make this two words?

I know this probably clashes with the specified design. Just to back up the suggestion a little bit, I googled `"screen readers"` and got 80 million results, and then googled `"screenreaders"` and got 200K results. In other words, the term screen readers as two words is 400 times as common.

File: client/components/DataManagement/CoveredAtListDialogWithButton/index.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                {renderButtonLabel()}
+            </Button>
+            <BasicModal
+                title="Covered AT"
+                show={show}
+                closeButton={true}
+                cancelButton={false}
+                handleClose={handleClose}
+                content={renderDialogContent()}
+            />
```
Comment (alflennik) [inline client/components/DataManagement/CoveredAtListDialogWithButton/index.jsx:48]: Would it be possible to reduce slightly the amount of padding in the modal?

**Review by alflennik — CHANGES_REQUESTED** [review]
I tested it in NVDA and it worked great. Nice.

**Review by howard-e — APPROVED** [review]
Looks good!

**Review by alflennik — APPROVED** [review]
Looks great!

**Comment by stalgiag** [comment]
Closing this as I don't think it's relevant to the current UI anymoore

**Pull request closed without merging**